SimpleJavaProject/src/main/java/ana_raetcaia/IfElseMain.java

```diff
@@ -0,0 +1,38 @@
+package ana_raetcaia;
+
+import java.util.Scanner;
+
+public class IfElseMain {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a num: ");
+        int num = scanner.nextInt();
+
+        if (num > 0) {
```

Comment:
Multiple if else should be replaced by switch everityme

Reply:
Fixed.

Reply:
How can I replace the if statement with the switch, which does not accept expressions as a range of some values (a>b, a<b, etc. that are listed in the task)? or is there another solution that includes a switch and range?

Reply:
devide score by 10

Reply:
Fixed